### PR TITLE
[WIP]Remove jquery-ui

### LINKED ISSUE #3007: Remove jquery-ui from batch 2 of scripts
Remove jquery-ui from following scripts:
interface/drugs/add_edit_drug.php
interface/fax/fax_dispatch.php
interface/forms/LBF/new.php
interface/forms/eye_mag/SpectacleRx.php
interface/forms/eye_mag/a_issue.php
interface/forms/eye_mag/view.php

### PR DESCRIPTION
**Pull request by im-Amitto**
Fixes #3007

### CODE REVIEW

**Comment by tywrenn** [comment]
@im-Amitto I love the BS4 replacements here 👍

**Comment by bradymiller** [comment]
On testing:

In eye form, getting following js console error:
image

To get to eye form:
Encounter gui->Clinical->Eye Exam

**Comment by bradymiller** [comment]
hi @im-Amitto ,
Tested scripts and only noted bug posted above,

**Comment by im-Amitto** [comment]
@bradymiller is their any library available in project for drag/drop functionalities which we can use inplace of drag/drop of jquery-ui?

**Comment by bradymiller** [comment]
hi @im-Amitto , Check out library/js/utility.js , which is included in all script. This is where @sjpadgett incorporated interact.js for drag and resize stuff.

**Comment by im-Amitto** [comment]
@bradymiller Can't find anything that i can use inplace of droppable.

File: interface/fax/fax_dispatch.php [hunk 1]
```diff
@@ -751,35 +750,35 @@ function allCheckboxes(issel) {
 
 <p><input type='checkbox' name='form_cb_forward' value='1'
  onclick='return divclick(this,"div_forward");' />
-<b><?php echo xlt('Forward Pages via Fax'); ?></b></p>
+<strong><?php echo xlt('Forward Pages via Fax'); ?></strong></p>
```
Comment (tywrenn) [inline interface/fax/fax_dispatch.php]: The `<p>` element that wraps this should have the class of `font-weight-bold` and the `<strong>` element then can be removed.

File: interface/fax/fax_dispatch.php [hunk 2]
```diff
@@ -810,7 +809,7 @@ function allCheckboxes(issel) {
 <input type='button' value='<?php echo xla('Clear All'); ?>' onclick='allCheckboxes(false)' />
 </p>
 
-<p><br /><b><?php echo xlt('Please select the desired pages to copy or forward:'); ?></b></p>
+<p><br /><strong><?php echo xlt('Please select the desired pages to copy or forward:'); ?></strong></p>
```
Comment (tywrenn) [inline interface/fax/fax_dispatch.php]: The `<p>` element should have the `font-weight-bold` class and the `<strong>` tag can then be removed

File: interface/fax/fax_dispatch.php [hunk 3]
```diff
@@ -463,7 +463,7 @@ function mergeTiffs()
 <html>
```
Comment (tywrenn) [inline interface/fax/fax_dispatch.php]: Please add a todo for this file to modernize it as it is using table elements that aren't responsive here.

File: interface/fax/fax_dispatch.php [hunk 4]
```diff
@@ -615,43 +614,43 @@ function allCheckboxes(issel) {
...
 
 <form method='post' name='theform'
  action='fax_dispatch.php?<?php echo ($mode == 'fax') ? 'file' : 'scan'; ?>=<?php echo attr_url($filename); ?>&csrf_token_form=<?php echo attr_url(CsrfUtils::collectCsrfToken()); ?>' onsubmit='return validate()'>
 <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
 
 <p><input type='checkbox' name='form_cb_copy' value='1'
- onclick='return divclick(this,"div_copy");' />
-<b><?php echo xlt('Copy Pages to Patient Chart'); ?></b></p>
+onclick='return divclick(this,"div_copy");' />
+<strong><?php echo xlt('Copy Pages to Patient Chart'); ?></strong></p>
```
Comment (tywrenn) [inline interface/fax/fax_dispatch.php]: The `<p>` element should have the `font-weight-bold` class and the `<strong>` tag can then be removed

File: interface/forms/eye_mag/view.php [hunk 5]
```diff
@@ -2621,14 +2621,14 @@ class="PRIORS_class PRIORS"><i class="fa fa-spinner fa-spin"></i>
                                                         <br />
                                                         <table>
                                                            <tr>
-                                                                <td style="text-align:center;"><?php echo xlt('R{{right}}'); ?></td>
+                                                                <td class='text-center'><?php echo xlt('R{{right}}'); ?></td>
                                                                 <td style="border-right:1pt solid black;border-bottom:1pt solid black;text-align:right;">
```
Comment (tywrenn) [inline interface/forms/eye_mag/view.php]: Convert the style tag of this `<td>` element to the following classes: `border-right border-bottom border-dark text-right`

File: interface/forms/eye_mag/view.php [hunk 6]
```diff
@@ -2621,14 +2621,14 @@ class="PRIORS_class PRIORS"><i class="fa fa-spinner fa-spin"></i>
...
                                                                 <textarea id="ACT1CCDIST" name="ACT1CCDIST" class="ACT"><?php echo text($ACT1CCDIST); ?></textarea></td>
                                                                 <td style="border:1pt solid black;border-top:0pt;text-align:center;">
```
Comment (tywrenn) [inline interface/forms/eye_mag/view.php]: Convert the style tag of this `<td>` element to the following classes: `border border-top-0 border-dark text-center`

File: interface/forms/eye_mag/view.php [hunk 7]
```diff
@@ -2621,14 +2621,14 @@ class="PRIORS_class PRIORS"><i class="fa fa-spinner fa-spin"></i>
...
                                                                 <textarea id="ACT2CCDIST"  name="ACT2CCDIST"class="ACT"><?php echo text($ACT2CCDIST); ?></textarea></td>
                                                                 <td style="border-left:1pt solid black;border-bottom:1pt solid black;text-align:left;">
```
Comment (tywrenn) [inline interface/forms/eye_mag/view.php]: Convert the style tag of this `<td>` element to the following classes: `border-left border-bottom border-dark text-left`

File: interface/forms/eye_mag/view.php [hunk 8]
```diff
@@ -2660,14 +2660,14 @@ class="PRIORS_class PRIORS"><i class="fa fa-spinner fa-spin"></i>
                                                         <br />
                                                         <table>
                                                             <tr>
-                                                                <td style="text-align:center;"><?php echo xlt('R{{right}}'); ?></td>
+                                                                <td class='text-center'><?php echo xlt('R{{right}}'); ?></td>
                                                                 <td style="border-right:1pt solid black;border-bottom:1pt solid black;text-align:right;">
```
Comment (tywrenn) [inline interface/forms/eye_mag/view.php]: Convert the style tag of this `<td>` element to the following classes: `border-right border-bottom border-dark text-right`

File: interface/forms/eye_mag/view.php [hunk 9]
```diff
@@ -2660,14 +2660,14 @@ class="PRIORS_class PRIORS"><i class="fa fa-spinner fa-spin"></i>
...
                                                                 <textarea id="ACT1SCNEAR" name="ACT1SCNEAR" class="ACT"><?php echo text($ACT1SCNEAR); ?></textarea></td>
                                                                 <td style="border:1pt solid black;border-top:0pt;text-align:center;">
```
Comment (tywrenn) [inline interface/forms/eye_mag/view.php]: Convert the style tag of this `<td>` element to the following classes: `border border-top-0 border-dark text-center`

File: interface/forms/eye_mag/view.php [hunk 10]
```diff
@@ -2660,14 +2660,14 @@ class="PRIORS_class PRIORS"><i class="fa fa-spinner fa-spin"></i>
...
                                                                 <textarea id="ACT2SCNEAR"  name="ACT2SCNEAR"class="ACT"><?php echo text($ACT2SCNEAR); ?></textarea></td>
                                                                 <td style="border-left:1pt solid black;border-bottom:1pt solid black;text-align:left;">
```
Comment (tywrenn) [inline interface/forms/eye_mag/view.php]: Convert the style tag of this `<td>` element to the following classes: `border-left border-bottom border-dark text-left`

File: interface/forms/eye_mag/view.php [hunk 11]
```diff
@@ -2699,14 +2699,14 @@ class="PRIORS_class PRIORS"><i class="fa fa-spinner fa-spin"></i>
                                                         <br />
                                                         <table>
                                                             <tr>
-                                                                <td style="text-align:center;"><?php echo xlt('R{{right}}'); ?></td>
+                                                                <td class='text-center'><?php echo xlt('R{{right}}'); ?></td>
                                                                 <td style="border-right:1pt solid black;border-bottom:1pt solid black;text-align:right;">
```
Comment (tywrenn) [inline interface/forms/eye_mag/view.php]: Convert the style tag of this `<td>` element to the following classes: `border-right border-bottom border-dark text-right`

File: interface/forms/eye_mag/view.php [hunk 12]
```diff
@@ -2699,14 +2699,14 @@ class="PRIORS_class PRIORS"><i class="fa fa-spinner fa-spin"></i>
...
                                                                 <textarea id="ACT1CCNEAR" name="ACT1CCNEAR" class="ACT"><?php echo text($ACT1CCNEAR); ?></textarea></td>
                                                                 <td style="border:1pt solid black;border-top:0pt;text-align:center;">
```
Comment (tywrenn) [inline interface/forms/eye_mag/view.php]: Convert the style tag of this `<td>` element to the following classes: `border border-top-0 border-dark text-center`

File: interface/forms/eye_mag/view.php [hunk 13]
```diff
@@ -2699,14 +2699,14 @@ class="PRIORS_class PRIORS"><i class="fa fa-spinner fa-spin"></i>
...
                                                                 <textarea id="ACT2CCNEAR"  name="ACT2CCNEAR"class="ACT"><?php echo text($ACT2CCNEAR); ?></textarea></td>
                                                                 <td style="border-left:1pt solid black;border-bottom:1pt solid black;text-align:left;">
```
Comment (tywrenn) [inline interface/forms/eye_mag/view.php]: Convert the style tag of this `<td>` element to the following classes: `border-left border-bottom border-dark text-left`

**Comment by bradymiller** [comment]
hi @im-Amitto , That library/js/utility.js using the [interact.js](https://interactjs.io/) package. Are we able to use that package to get what is needed?

**Comment by tywrenn** [comment]
@im-Amitto Still working on this?

**Comment by im-Amitto** [comment]
@tywrenn I actually shifted my focus on flutter openemr flutter but I will try to fix it in within 48 hours if it doesn't work let's close it as I can't abandon the flutter repo for too long.

**Pull request closed without merging**